aocd/get.py

```diff
@@ -73,7 +81,7 @@ def current_day():
     return day
 
 
-def get_day_and_year():
+def get_day_and_year() -> tuple[int, int]:
```

Comment:
It's currently possible for this to return (int, None) but we were changing that in the other PR.

Wasn't pyright able to detect that possibility?

Reply:
It won't tell you when it only checks the public API annotations. At some point I was running it in strict mode through the whole codebase but that got really noisy. I can't quite remember what happened here, I could have sworn I had this as `tuple[int, int | None]` before.